src/utils.ts

```diff
@@ -22,18 +22,18 @@
  * This is specifically useed by the Kibana proxy re-routing.
  * It also handles trailing slashes in tileApiUrl and fileApiUrl parameters.
  */
-export function toAbsoluteUrl (host, path) {
+export const toAbsoluteUrl = (host: string | undefined, path: string): string => {
```

Comment:
I wouldn't replace this with a lambda. the 'function'-keyword makes it readable. lambdas are good for when `this` scoping is important, otherwise it's more indirection.